crates/core/src/blob/packer.rs

```diff
@@ -91,6 +91,8 @@ impl PackSizer {
 
     /// Computes the size of the pack file.
     #[must_use]
+    // TODO!: IMPORTANT: Evaluate deeply if this is really fine and what we want
```

Comment:
This needs further evaluation

Reply:
The cast actually shouldn't pose any problems.
`current_size` is `u64`, the maximum value is `2^64-1`.
`isqrt(2^64-1) = 2^32-1` which fits into a `u32`.